src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -97,25 +97,23 @@ namespace Platform
         }
         // 2. Try {${exeDir},${cwd},/}/{data,standard system app directories}
         // exeDir should come first to allow installing into build dir
-        std::vector<std::string> prefixes;
-        auto exePath = Platform::GetCurrentExecutablePath();
-        prefixes.push_back(Path::GetDirectory(exePath));
-        prefixes.push_back(GetCurrentWorkingDirectory());
-        prefixes.push_back("/");
-        static const char* SearchLocations[] = {
+        // clang-format off
```

Comment:
I didn't really like the way clang-format decided to join the lines for these, looks weird, that's why the clang-format off, it would just spit em on a single line

Reply:
Try adding a comma to the last element, that sometimes does the job.

Reply:
I'm on it, it might work indeed, recall having done a similar thing

Reply:
back with some insights, adding a comma at the last element solves it for the first case, but weirdly enough the second one, while already having a comma, still gets put on one line